pure.zsh

```diff
@@ -186,6 +186,11 @@ prompt_pure_precmd() {
 		psvar[12]="${VIRTUAL_ENV:t}"
 		export VIRTUAL_ENV_DISABLE_PROMPT=12
 	fi
+	# Check if a conda environment is active and display it's name
+	if [[ -n $CONDA_DEFAULT_ENV ]]; then
+		psvar[12]="${CONDA_DEFAULT_ENV//[$'\t\r\n']}"
+		export VIRTUAL_ENV_DISABLE_PROMPT=12
```

Comment:
Why set `VIRTUAL_ENV_DISABLE_PROMPT` here? It has no effect on conda. Also, what if both conda and virtualenv are active at the same time? In that case, the conda `psvar[12]` would override the virtualenv which could be confusing.

Reply:
You're right, `VIRTUAL_ENV_DISABLE_PROMPT` should not be altered.
```suggestion
```